specification/programmableconnectivity/ProgrammableConnectivity.Management/OperatorApiConnection.tsp

```diff
@@ -125,43 +158,379 @@ union AccountType {
 }
 
 /**
- * Details about the Application that would use the Operator's Network APIs.
+ * Model defining how received data is processed.
  */
-model ApplicationProperties {
+@TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+model DataProcessing {
   /**
-   * Name of the application. Example: Contoso App.
+   *  The operation that is performed on the data received.
    */
-  name?: string;
+  processingOperation: ProcessingOperation;
 
   /**
-   * Description of the application.
+   * The contexts in which the data is processed.
    */
-  applicationDescription?: string;
+  contexts: Context[];
 
   /**
-   * The category that describes the application.
+   * The duration or temporal limitation of the data processing.
    */
-  applicationType?: string;
+  duration: Duration;
 
   /**
-   * Legal name of the organization owning the application.
+   * The frequency of the data processing.
+   */
+  frequency: Frequency;
+
+  /**
+   * The countries/regions the data transits through.
+   */
+  @OpenAPI.extension("x-ms-identifiers", ["countryCode"])
+  transitRegions: DataRegions[];
+
+  /**
+   * The countries/regions the data is stored in.
+   */
+  @OpenAPI.extension("x-ms-identifiers", ["countryCode"])
+  storageRegions: DataRegions[];
+}
+
+/**
+ * Model defining the information required about the countries/regions the data transits through or is stored in.
+ */
+@TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+model DataRegions {
+  /**
+   * The country code of the country/region the data transits through or is stored in.
    */
-  legalName?: string;
+  @pattern("[A-Z]{2}", "Provide 2 letter country code")
+  countryCode: string;
 
   /**
-   * A description of the organization owning the application.
+   * The commercial organization who is managing the data.
    */
-  organizationDescription?: string;
+  commercialOrganization?: string;
 
   /**
-   * Unique Tax Number for the user's organization in the country/region the APC Gateway is being purchased.
+   * Flag to indicate if commercial activity applies to this data management.
    */
-  taxNumber?: string;
+  commercialActivity: boolean;
 
   /**
-   * Email address of the Privacy contact or Data Protection officer of the organization.
+   * URL to the organization or governmental data privacy framework.
    */
-  privacyContactEmailAddress?: string;
+  dataPrivacyFrameworkUrl: string;
+}
+
+/**
+ * Valid values for the frequency based on W3C Data Privacy Vocabulary v2 https://w3c.github.io/dpv/2.0/dpv/.
+ */
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
```

Comment:
Needed?

Reply:
As above, these are in our API because we forward them to another, industry-standard, API. There is no extra description in the industry-standard API definition so we don't want to add information; therefore, any documentation here would just add a description that is the same as the value which isn't useful. Therefore, I decided to suppress documentation required.

Reply:
Resolving this, but see my other comment.